init.go

```diff
@@ -150,17 +150,7 @@ func (a *initFnCmd) init(c *cli.Context) error {
 		}
 	}
 
-	const runHeader = `
-        ______
-       / ____/___
-      / /_  / __ \
```

Comment:
@rdallman why remove this? want for fn init.

Reply:
see parent

Reply:
just obfuscates the 'your route has been updated' and usually when commands are loud i think something went wrong. this one is especially loud.

Reply:
ascii text in download command seems like what's really wanted. instead of bashing everyone's heads with 10 lines of ascii text every time they make a new function. i realize this is the Thiel way, but seems like a show once vs show all the time kind of thing. not much for fru fru clis myself though...

Reply:
I'm with @rdallman (same comment I had when you added this in @carimura). Maybe in a less common command this would make sense, but it's annoying a fairly common one like init. +1 on install/download, much better place for it.

Reply:
okie dokie.